### ISSUE TITLE
Lack of "row" on search.html results

### ISSUE THREAD

**Issue by data-digger** [cocteauuk/thenude#8]
Regarding those lines of code:
https://github.com/cocteauuk/thenude/blob/73c5dca9510f749d2e43e7070994f48e8753520a/search.html#L124-L126
```            
    <!-- ==================== SEARCH TABS ==================== -->
     <div class="top-buffer">
                <div class="col-md-12">
````

Shouldn't the `<div class="top-buffer">` contain "row" class as well? Otherwise the float seems to slip to next divs

**Comment by cocteauuk** [comment]
The only function of .row is to give negative padding either side of the group of columns or content. I'm not sure what can be the issue, is there any way that you could let me try it in the browser? can you upload something for me to see the issue?

But if you think adding .row might help, by all means let me know so I add it.

Thanks

**Comment by data-digger** [comment]
Fixed in 51d5b8867cf55925295f2d7b657344f8b77972ad . `row` class also clears float. I used `clearfix` class instead.
